cli/internal/cmd/configgenerate.go

```diff
@@ -221,3 +230,30 @@ func parseAttestationFlag(flags *pflag.FlagSet) (variant.Variant, error) {
 
 	return attestationVariant, nil
 }
+
+func parseTagsFlags(flags *pflag.FlagSet) (cloudprovider.Tags, error) {
+	tagsString, err := flags.GetString("tags")
+	if err != nil {
+		return nil, fmt.Errorf("getting tags flag: %w", err)
+	}
+	tagsString = strings.ReplaceAll(tagsString, " ", "")
+
+	// no tags given
+	if tagsString == "" {
+		return nil, nil
+	}
+
+	tagsStringSplit := strings.Split(tagsString, ",")
+
+	tags := make(cloudprovider.Tags)
+	for _, tag := range tagsStringSplit {
+		tagSplit := strings.Split(tag, "=")
+		if len(tagSplit) != 2 {
+			return nil, fmt.Errorf("wrong format of tags")
+		}
```

Comment:
```suggestion
		if len(tagSplit) != 2 {
			return nil, fmt.Errorf("wrong format of tags: expected \"key=value\", got: %q", tag)
		}
```

Only use `fmt.Errorf` when the error requires format strings, otherwise just use `errors.New`.
In this case, I think it helps to tell the user why the format is wrong